clients/cloud/python/producer.py

```diff
@@ -0,0 +1,88 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+import json
+import ccloud_lib
+
+
+
+if __name__ == '__main__':
+
+    # Initialization
+    args = ccloud_lib.parse_args()
+    config_file = args.config_file
+    topic = args.topic
+    conf = ccloud_lib.read_ccloud_config(config_file)
+
+    # Create Producer instance
+    p = Producer({
+           'bootstrap.servers': conf['bootstrap.servers'],
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': conf['sasl.username'],
+           'sasl.password': conf['sasl.password']
+    })
+
+    # Create topic if needed
+    # Examples of additional admin API functionality: https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/adminapi.py
+    a = AdminClient({
+           'bootstrap.servers': conf['bootstrap.servers'],
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': conf['sasl.username'],
+           'sasl.password': conf['sasl.password']
+    })
+    fs = a.create_topics([NewTopic(topic, num_partitions=1, replication_factor=3)])
+    for topic, f in fs.items():
+        try:
+            f.result()  # The result itself is None
+            print("Topic {} created".format(topic))
+        except Exception as e:
+            # Continue if error code TOPIC_ALREADY_EXISTS, which may be valid if topic exists
+            if e.args[0].code() != KafkaError.TOPIC_ALREADY_EXISTS:
+              print("Failed to create topic {}: {}".format(topic, e))
+
+    # Optional per-message on_delivery handler (triggered by poll() or flush())
+    # when a message has been successfully delivered or
+    # permanently failed delivery (after retries).
+    def acked(err, msg):
+        """Delivery report handler called on successful or failed delivery of the message."""
+        if err is not None:
+            print("failed to deliver message: {0}".format(err.str()))
+        else:
+            print("Successfully produced record to topic {} partition [{}] @ offset {}".format(msg.topic(), msg.partition(), msg.offset()))
+
+    for n in range(10):
+      record_key = "alice"
+      record_value = json.dumps({'count': n})
+      print ("Preparing to produce record: {} \t {}".format(record_key, record_value))
+      p.produce(topic,key=record_key,value=record_value,on_delivery=acked)
+      # p.poll(0) serves delivery reports (on_delivery) from previous produce() calls.
```

Comment:
remove 0, might give people the idea that it only does this if a timeout of 0 is provided